src/ast/OpMinus.js

```diff
@@ -39,6 +39,8 @@ function createNode(position, left, right) {
     var node = SpelNode.create('op-minus', position, left, right);
 
     node.getValue = function (state) {
+        if (!right)
+            return - left.getValue(state);
```

Comment:
So this would evaluate something like `-5`? I'm surprised that doesn't result in left being null and right being `5`. Could you just wrap the return in braces? Same with the OpPlus changes.
```suggestion
        if (!right) {
            return - left.getValue(state);
        }
```